tests/wpunit/Tribe/DependencyTest.php

```diff
@@ -162,22 +162,22 @@ public function dependency_matrix() {
 			],
 		];
 
-		yield 'All deps ok' => [
-			[
-				'one'   => array_merge( $one, [
-					'should_initialize' => true,
-					'failure_message'   => 'Plugin One should activate.',
-				] ),
-				'two'   => array_merge( $two, [
-					'should_initialize' => true,
-					'failure_message'   => 'Plugin Two should activate: its version satisfies One\'s requirements.',
-				] ),
-				'three' => array_merge( $three, [
-					'should_initialize' => true,
-					'failure_message'   => 'Plugin Three should activate: its version satisfies One\'s requirements.',
-				] ),
-			],
-		];
+//		yield 'All deps ok' => [
+//			[
+//				'one'   => array_merge( $one, [
+//					'should_initialize' => true,
+//					'failure_message'   => 'Plugin One should activate.',
+//				] ),
+//				'two'   => array_merge( $two, [
+//					'should_initialize' => true,
+//					'failure_message'   => 'Plugin Two should activate: its version satisfies One\'s requirements.',
+//				] ),
+//				'three' => array_merge( $three, [
+//					'should_initialize' => true,
+//					'failure_message'   => 'Plugin Three should activate: its version satisfies One\'s requirements.',
+//				] ),
+//			],
+//		];
```

Comment:
remove commented out code?

Reply:
I had it commented during debug, it should not be commented out, thanks!